src/main/java/com/algorand/algosdk/v2/client/algod/RawTransaction.java

```diff
@@ -0,0 +1,65 @@
+package com.algorand.algosdk.v2.client.algod;
+
+import java.io.IOException;
+
+import com.algorand.algosdk.v2.client.connect.Client;
+import com.algorand.algosdk.v2.client.connect.Query;
+import com.algorand.algosdk.v2.client.connect.QueryData;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+
+/**
+ * /v2/transactions 
+ */
+public class RawTransaction extends Query {
+	private String rawtxn;
+
+	private boolean rawtxnIsSet;
+
+	public RawTransaction(Client client) {
+		super(client);
+	}
+
+	/**
+	 * The byte encoded signed transaction to broadcast to network 
+	 */
+	public RawTransaction setRawtxn(String rawtxn) {
```

Comment:
We could also have helpers which allow submitting a `SignedTransaction` directly, or `List<SignedTransaction>`, etc.